libredex/Trace.h

```diff
@@ -91,6 +91,7 @@
   TM(PGR)            \
   TM(PM)             \
   TM(POST_LOWERING)  \
+  TM(PROGUARD)       \
```

Comment:
Some of the existing code uses PGR to denote ProGuard trace statement, probably makes sense to reuse that?

Reply:
Whoa didn't realize that's what that stood for! Thanks for pointing that out! I'll get a new rev up soon!